js/commit-page.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Javascript for project commit pages to update the status and build the left sidebar
+ */
+
+$(document).ready(function(){
+  var url_parts = window.location.href.split('?')[0].split('/');
+  var parent_url = url_parts.slice(0,-2).join('/');
+  var myCommitId = url_parts.slice(-1);
+
+  var request;
+  if(window.XMLHttpRequest)
+    request = new XMLHttpRequest();
+  else
+    request = new ActiveXObject("Microsoft.XMLHTTP");
```

Comment:
Understandable. Just took this bit from stack overflow of someone also needing to see if a page existed. Will use better way with Ajax. May just try to load the commit's build_log.json as I did this page's.